Multisig/UI/ClaimToken/ReviewClaimTokensHeaderCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ReviewClaimTokensHeaderCell.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 07.09.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class ReviewClaimTokensHeaderCell: UITableViewCell {
+    @IBOutlet private weak var amountView: TokenInfoView!
+    @IBOutlet private weak var delegateView: AddressInfoView!
+    @IBOutlet private weak var fromView: AddressInfoView!
+    @IBOutlet private weak var toView: AddressInfoView!
+
+    var showsDelegate: Bool = true {
```

Comment:
Maybe instead of this we could hide delegate view always until setDelegate called with value ?

Reply:
can you explain your reasoning a bit, i.e. why this would be better?

Reply:
with the current implementation in order to present delegate we need to set it first then set showsDelegate, with my suggestion by just setting the delegate it should be showed